ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController.swift

```diff
@@ -127,7 +128,9 @@ class AvatarDemoController: DemoTableViewController {
                 cell.contentView.bottomAnchor.constraint(equalTo: avatarContentView.bottomAnchor)
             ])
 
-            cell.backgroundConfiguration?.backgroundColor = self.isUsingAlternateBackgroundColor ? Colors.tableCellBackgroundSelected : Colors.tableCellBackground
```

Comment:
This line of code would not modify the background configuration, which is why the change here was made.